### DISCUSSION TITLE
Sonoff 4CHPRO R3 433Mhz Control

### DISCUSSION THREAD

**Discussion by felix-196** [arendst/Tasmota#20063] [Q&A]
When flashing Tasmoto on Sonoff 4CHPRO rev R3, is Remote 433Mhz still supported?

**Comment by sfromis** [comment]
https://templates.blakadder.com/sonoff_4CHPROR3.html says 
> 4CHPROR3 supports [RF remotes](https://templates.blakadder.com/sonoff_RM433) independent from Tasmota, it is managed by a secondary MCU.

Hence, Tasmota should make no difference to what you can and should do around the remote.

**Comment by barbudor** [comment]
Already discussed many time
And documented
https://tasmota.github.io/docs/devices/Sonoff-4CH-Pro/#sonoff-4ch-pro-r3

**Comment by felix-196** [comment]
4CHPROR3 has no secondary MCU. Doku seems to be wrong.

4CHPROR1 got secondary MCU



Von: sfromis ***@***.***
Gesendet: Sonntag, 19. November 2023 20:01
An: arendst/Tasmota
Cc: felix-196; Author
Betreff: Re: [arendst/Tasmota] Sonoff 4CHPRO R3 433Mhz Control (Discussion #20063)



https://templates.blakadder.com/sonoff_4CHPROR3.html says

4CHPROR3 supports RF remotes <https://templates.blakadder.com/sonoff_RM433>  independent from Tasmota, it is managed by a secondary MCU.

Hence, Tasmota should make no differencer to what you can and should do around the remote.

—
Reply to this email directly, view it on GitHub <https://github.com/arendst/Tasmota/discussions/20063#discussioncomment-7612846> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXUL3X7NPNLQPW23AQSQJ2LYFJJOVAVCNFSM6AAAAAA7R5Q4WGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMJSHA2DM> .
You are receiving this because you authored the thread.  <https://github.com/notifications/beacon/AXUL3XYS3XVIQYDMGJLDKFTYFJJOVA5CNFSM6AAAAAA7R5Q4WGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAOQU24.gif> Message ID: ***@***.***>



--
Diese E-Mail wurde von Avast-Antivirussoftware auf Viren geprüft.
www.avast.com

**Reply by barbudor** [reply]
Not owning such device I can't help further but if you can tracks the connections on the PCB and revert your findings, it would be possible to advise, and update the docs

**Reply by sfromis** [reply]
The photo on the templates page I linked clearly says "SONOFF_4CH_PRO_R3" on the PCB, and a workaround to bypass the extra MCU. Are you saying that there is a different board revision now?

**Reply by sfromis** [reply]
This page suggests that the non-"pro" Sonoff 4CH R3 does **not** have the extra MCU, but the "pro" version does.
https://notenoughtech.com/home-automation/sonoff-4ch-4chpro-joins-r3-revision/

**Reply by felix-196** [reply]
I got a brandnew Pro3 here. Marking on PCB is SONOFF_4CH_PRO_R3 and there is only 1 MCU on board.
How can I attache picture?

**Reply by sfromis** [reply]
I'm usually copy/pasting a selection from an image editor or screenshot tool into the page for the forum. There is also an "Attach files" paperclip button on the toolbar when I write a post.

Anyway, it does sound like a new hardware revision, and what you describe sounds much more similar to the non-"pro" one, but still to be basically explored as a "new device" to see how it works.

**Reply by felix-196** [reply]
Now I got it ;)
![4CH_PRO_R3](https://github.com/arendst/Tasmota/assets/99139039/a20a8338-987c-4623-82b0-dba2d26fa7c4)

**Reply by sfromis** [reply]
And you even connected a resistor to the extra MCU on the board :smile: 

Looks like the usual EFM8BB1, which might be connected via serial to the ESP. Or not.

**Comment by felix-196** [comment]
That IC is not an extra MCU. It is a SYN470R ASK Receiver.
I tryied to fix/improve the output ghost switching issue with the resistor. See SYN470R datasheet, data squelch 
I still not found an extra MCU ;)

**Reply by sfromis** [reply]
Trace the wiring.

**Reply by jamesshannon** [reply]
> That IC is not an extra MCU. It is a SYN470R ASK Receiver. I tryied to fix/improve the output ghost switching issue with the resistor. See SYN470R datasheet, data squelch I still not found an extra MCU ;)

@felix-196 Were you able to trace the output of the SYN470R? According to the web, it "is a true “antenna-in to data-out” monolithic device" so presumably it's just a matter of figuring out which 8255 pin it outputs to and the baud rate?
